### DISCUSSION TITLE
Merging multiple test result

### DISCUSSION THREAD

**Discussion by balamurugansanthanaraj** [stryker-mutator/stryker-net#1237] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Can you suggest, how to merge multiple result files as on result HTML file?

**Describe the solution you'd like**

**Describe alternatives you've considered**
Not yet thought

**Additional context**

**Comment by rouke-broersma** [comment, marked as answer]
Hi!

You can use this blog post as a starting point to do that: https://rajbos.github.io/blog/2019/10/11/Parallelizing-a-long-Stryker-Run-in-Azure-DevOps

**Reply by balamurugansanthanaraj** [reply]
Thanks for sharing the article. Indeed very useful
